modules/openapi-generator/src/test/java/org/openapitools/codegen/java/spring/SpringCodegenTest.java

```diff
@@ -877,13 +892,13 @@ public void testConfigFileGeneration() {
             tmpFile = s.getTemplateFile();
             desFile = s.getDestinationFilename();
```

Comment:
Is there any value in testing the contents of the file?

Can always be added later if needed to validate any future bug fixes

Reply:
Yeah - just comparing to the test above which actually runs the generator and then validates the file content 

```
        assertFileContains(Paths.get(outputPath + "/src/main/java/org/openapitools/api/GetApi.java"),
            "@RequestParam(value = \"testParameter1\", required = false, defaultValue = \"BAR\")",
            "@RequestParam(value = \"TestParameter2\", required = false, defaultValue = \"BAR\")");
```

But that might be overkill to do that for this test, at this point anyway.

Reply:
I saw your example below - thanks for showing it! I didn't spot that when I was thinking about it. I think I agree it's a little overkill for this feature, but definitely will keep this in mind for the future. As long as you have no existing concerns on this thread- I'll let you resolve the conversation when you get the chance :). I appreciate the discussion!